include/envoy/network/connection.h

```diff
@@ -19,13 +19,12 @@ class Dispatcher;
 namespace Network {
 
 /**
- * Events that occur on a connection. Maybe be combined.
+ * Events that occur on a connection.
  */
-class ConnectionEvent {
-public:
-  static const uint32_t RemoteClose = 0x1;
-  static const uint32_t LocalClose = 0x2;
-  static const uint32_t Connected = 0x4;
+enum ConnectionEvent {
```

Comment:
Ooh, good call.  Updated this, and a bunch of tests that subsequently failed to compile.